### PR TITLE
Bug Squashing

### PR DESCRIPTION
**Pull request by DXCanas**
## Summary

Squashing bugs, 

https://trello.com/c/oN4jJXxi/370-users-bug-user-create-modal-refers-to-fullname-in-template-but-firstname-and-lastname-in-data

https://trello.com/c/W7c6dWfO/365-users-bug-add-user-dialog-retains-contents-from-previously-added-user

### CODE REVIEW

**Comment by DXCanas** [comment]
Had some conflicts, with master. Pushed up fix.

File: kolibri/plugins/management/assets/src/vue/user-page/index.vue [hunk 1]
```diff
@@ -51,14 +51,14 @@
...
-          <th scope="row" class="table-cell">
+          <td scope="row" class="table-cell">
             {{user.full_name}}
 
             <!-- Logic for role tags -->
             <span class="user-role" v-for="role in user.roles">
               {{role.kind | capitalize}}
             </span>
-          </th>
+          </td>
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/vue/user-page/index.vue]: this was a change that @radinamatic had requested - was it not working as expected?

File: kolibri/plugins/management/assets/src/actions.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ function _userState(data) {
 function createUser(store, payload, role) {
   const FacilityUserModel = FacilityUserResource.createModel(payload);
   const newUserPromise = FacilityUserModel.save(payload);
-  newUserPromise.then((model) => {
+  return newUserPromise.then((model) => {
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/actions.js:43]: why are we returning a promise from an action?

Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/actions.js:43]: oh I see - you're using it as a promise below

File: kolibri/plugins/management/assets/src/vue/user-page/index.vue (hunk 1 above)
Comment (radinamatic) [inline kolibri/plugins/management/assets/src/vue/user-page/index.vue]: Works OK in current master. 
@DXCanas Check [Associate the Data Cells with the Appropriate Headers](http://webaim.org/techniques/tables/data#headers) for reference.

**Pull request merged**